include/boost/url/detail/pct_encoding.hpp

```diff
@@ -44,6 +34,37 @@ key_equal_encoded(
     string_view plain_key,
     string_view encoded) noexcept;
 
+BOOST_URL_DECL
+int
+pct_decode_compare_unchecked(
+    string_view lhs,
+    string_view rhs) noexcept;
```

Comment:
Are these called from user code?

Reply:
No. Removing `BOOST_URL_DECL`, right? I replicated that from `key_equal_encoded` at first but now removed it.

Reply:
`key_equal_encoded` might be needed for users at some point which is why I might have exported it, but I probably decided to just wait until someone needed it.